test/users/csep524/userDefReduceSlim.good

```diff
@@ -3,5 +3,5 @@ userDefReduceSlim.chpl:23: error: unresolved call 'type mostFrequent.mySet'
 userDefReduceSlim.chpl:9: note: this candidate did not match: mostFrequent.mySet
 userDefReduceSlim.chpl:23: note: because method call receiver is a type
 userDefReduceSlim.chpl:9: note: but is passed to non-type formal 'this: borrowed mostFrequent'
-  $CHPL_HOME/modules/internal/ChapelReduce.chpl:65: called as (mostFrequent(int(64))).combine(partner: owned mostFrequent(int(64)))
+  $CHPL_HOME/modules/internal/ChapelReduce.chpl:67: called as (mostFrequent(int(64))).combine(partner: owned mostFrequent(int(64)))
```

Comment:
Consider linking the standard prediff for ignoring standard module line numbers.

Reply:
```
ln -s ../../../util/test/prediff-obscure-module-linenos userDefReduceSlim.prediff
```